tests/parallel_floris_model_2_unit_test.py

```diff
@@ -0,0 +1,140 @@
+
+import copy
+import logging
+
+import numpy as np
+import pytest
+
+from floris import (
+    FlorisModel,
+    UncertainFlorisModel,
```

Comment:
I think would be good to add tests with WindRose and TimeSeries just to be sure

Reply:
Done